### PR TITLE
[Order][Promotion] Clean up

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | yes
| Related tickets | 
| License         | MIT

- [x] Order
- [x] OrderBundle
- [x] Promotion
- [x] PromotionBundle

### CODE REVIEW

File: src/Sylius/Bundle/OrderBundle/Resources/config/routing.yml [hunk 1]
```diff
@@ -3,20 +3,25 @@
...
     path: /add
-    defaults: { _controller: sylius.controller.order_item:addAction }
+    defaults:
+        _controller: sylius.controller.order_item:addAction
 
 sylius_cart_item_remove:
     path: /{id}/remove
-    defaults: { _controller: sylius.controller.order_item:removeAction }
+    defaults:
+        _controller: sylius.controller.order_item:removeAction
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/OrderBundle/Resources/config/routing.yml]: I think we decided to remove routing from bundles.

File: src/Sylius/Bundle/OrderBundle/Resources/config/validation/Adjustment.xml [hunk 2]
```diff
@@ -1,34 +0,0 @@
...
-                <option name="groups">sylius</option>
-            </constraint>
-            <constraint name="Type">
-                <option name="type">integer</option>
-                <option name="message">sylius.adjustment.amount.integer</option>
-                <option name="groups">sylius</option>
-            </constraint>
-        </property>
-    </class>
-</constraint-mapping>
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/OrderBundle/Resources/config/validation/Adjustment.xml]: Why removing validation?

File: src/Sylius/Bundle/OrderBundle/Resources/translations/flashes.en.yml [hunk 3]
```diff
@@ -4,8 +4,5 @@
 sylius:
     cart:
         cart_save_completed: The cart has been successfully updated.
-        cart_clear_completed: The cart has been successfully cleared.
         item_add_completed: Item has been added to cart.
         item_remove_completed: Item has been removed from cart.
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/OrderBundle/Resources/translations/flashes.en.yml]: These flashes are probably not used in bundle, have we decided where to put them?

File: src/Sylius/Bundle/OrderBundle/Resources/config/routing.yml (hunk 1 above)
Comment (GSadee) [inline src/Sylius/Bundle/OrderBundle/Resources/config/routing.yml]: Ok, I remove this file

File: src/Sylius/Bundle/OrderBundle/Resources/config/validation/Adjustment.xml (hunk 2 above)
Comment (GSadee) [inline src/Sylius/Bundle/OrderBundle/Resources/config/validation/Adjustment.xml]: I guess that we don't need validation for an adjustment as much as we don't need form type for this entity. Should I revert this change?

Comment (michalmarcinkowski) [inline src/Sylius/Bundle/OrderBundle/Resources/config/validation/Adjustment.xml]: We have CRUD for adjustments in ApiBundle, but we don't have tests for that, so it probably doesn't work now. Validation works not only for form types, but anytime the object is passed to the validator. Despite the fact that we don't verify this validation anywhere it is so simple and basic that I would leave it (we have such constraints on the database level as well).

**Comment by michalmarcinkowski** [comment]
Good job Grzesiu!
